src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -170,13 +170,27 @@ const defaultFeatureGateComponents = {
       ungatedFeatures: p.features,
       gatedFeatureGroups: [],
     }),
+  IntegrationDirectoryFeatures: p =>
+    p.children({
```

Comment:
@NisanthanNanthakumar Can we avoid duplicating the logic by creating a named function and using it for both keys? Getting rid of the legacy components will take some time because we actually use the component in places other than the integrations page such as the sentry app install page.